### ISSUE TITLE
expand readme.md

### ISSUE THREAD

**Issue by Caesurus** [CaptainBriot/pantheon-hermes#1]
Can you expand the readme to include examples of running things individually instead of needing ansible. Only mining farms will want to use ansible, the average user will want to run stand-alone.

**Comment by chonkerboi** [comment]
Good point. We should probably have a discussion about power management and overclocking, would the lambda user want them enabled by default?

**Comment by Caesurus** [comment]
No, overclocking and power management should be a very specific per card and you don't want to have default settings that could potentially damage cards without having tested them. I would say you even want to put a disclaimer in here somewhere that states this.

**Comment by chonkerboi** [comment]
Currently each card has a default (safe?) overclocking/undervolting configuration.

I have to think about how to configure the target and each GPU separately.
